## my environment running ArangoDB
I'm using the latest Arangodb of the respective release series:
- [ ] 2.8
- [ ] 3.0
- [ ] self compiled devel branch

On this operating system:
- [ ] DCOS on
  - [ ] AWS
  - [ ] Azzure
  - [ ] own infrastructure
- [ ] Linux [ ] other
  - [ ] Debian .deb
  - [ ] Ubuntu .deb
  - [ ] SUSE   .rpm
  - [ ] RedHat .rpm
  - [ ] Fedora .rpm
  - [ ] Gentoo
  - [ ] docker - official docker library
- [ ] Windows Version: 
- [ ] Macos Version:

### this is an AQL related issue:
[ ] I'm using graph features

I'm issuing AQL Via:

- [ ] webinterface with this browser:     running on this os:
- [ ] arangosh
- [ ] this Driver:

I've run `db._explain("<my aql query>")` and it didn't shed more light on this.
The AQL query in question is:

You can reproduce this using this dataset:

Please provied a way to create the dataset to run the above query on; either by a gist with an arangodump, or `db.collection.save({my: "values"}) statements. If it can be reproduced with one of the ArangoDB example datasets, its a plus.

### FOXX



### this is a Webinterface related issue:
I'm using the webinterface with this browser:     running on this os:
- [ ] authentication enabled?
- [ ] running cluster?

These are the steps to reproduce:
1) open the Browser on http://127.0.0.1:8529
2) log in as ...
3) to the database [ ] `_system` [ ] other: 
4) click to ...
...

Then this happenes: [Screenshot?] I would be expecting: 


### this is Installation related issue:
Describe what you did, what happened, what you would have expected to happen.
